src/content/pages/faq.mdx

```diff
@@ -47,11 +73,29 @@ Email us at [programme@europython.eu](mailto:programme@europython.eu) to tell us
 
 ## **Q. When do ticket sales start and what are the ticket prices like?**
 
-A. Ticket sales will be open soon! Stay tuned for more announcements.
+A. Ticket sales will be open on 25th of March! We have put a lot of work in keeping the ticket prices affordable. Check out our [tickets page](/tickets) for details.
+
+We encourage and trust you to pick a fair ticket tier that fits your personal circumstance. The money you spend will be put straight back to the community to support our diversity initiatives, Financial Aid Programme and other grants offered to the European Python community.
+
+## **Q. Can I attend EuroPython remotely?**
```

Comment:
We decided to just do YT streams for remote participants last year. This could be a bit misleading.

Reply:
Might be better if we remove it for now

Reply:
Done :hugs: